### PR TITLE
Update what-is-delegating.md

### PR DESCRIPTION
**Pull request by barriebyron**
typo fix and language love

### CODE REVIEW

File: content/articles/what-is-delegating.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ tags:
...
 
-In practice, delegating is done automatically when ATOM holders start staking. A single staking transaction needs to be sent for the whole proces of locking up ATOM and delegating to validator(s) via a [wallet]. Users will generally only be asked for the amount of ATOM to stake and the validator(s) they want to delegate to. The Wallet will take care of the rest (i.e. generating and sending the transaction).
+In practice, delegating is automatic when ATOM holders start staking. A single staking transaction starts the process of locking up ATOM and delegating to one or more validators. Transactions are performed using a [Wallet]. The transaction requires you to specify the amount of ATOM to stake and the validator or validators you want to delegate to. The Wallet takes care of the rest by generating and sending the transaction.
```
Comment (lovincyrus) [inline content/articles/what-is-delegating.md:17]: We can link the [Wallet] to `(/learn/faq/what-is-a-wallet)` page

File: content/articles/what-is-delegating.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ tags:
...
 
-Note that validators never obtain custody of the ATOM delegated to them. There is no risk for validators to "steal" their delegators ATOM. However, there is a risk for delegated ATOM to be slashed should the validator they are delegated to misbehave. See [What are the risks associated with Staking] for more. 
\ No newline at end of file
+Validators never obtain custody of the ATOM delegated to them. There is no risk for validators to "steal" their delegators ATOM. However, there is a risk for delegated ATOM to be slashed if the validator they are delegated to misbehaves. See [What are the risks associated with Staking].
```
Comment (lovincyrus) [inline content/articles/what-is-delegating.md:19]: We can also link the [What are the risks associated with Staking] to `(/learn/faq/what-are-the-risks-associated-with-staking)`

**Review by lovincyrus — CHANGES_REQUESTED** [review]

File: content/articles/what-is-delegating.md (hunk 1 above)
Comment (barriebyron) [inline content/articles/what-is-delegating.md:17]:
```suggestion
In practice, delegating is automatic when ATOM holders start staking. A single staking transaction starts the process of locking up ATOM and delegating to one or more validators. Transactions are performed using a [Wallet](/learn/faq/what-is-a-wallet). The transaction requires you to specify the amount of ATOM to stake and the validator or validators you want to delegate to. The Wallet takes care of the rest by generating and sending the transaction.
```

Comment (barriebyron) [reply content/articles/what-is-delegating.md:17]: I missed that!

File: content/articles/what-is-delegating.md (hunk 2 above)
Comment (barriebyron) [inline content/articles/what-is-delegating.md:19]:
```suggestion
Validators never obtain custody of the ATOM delegated to them. There is no risk for validators to "steal" their delegators ATOM. However, there is a risk for delegated ATOM to be slashed if the validator they are delegated to misbehaves. See [What are the risks associated with Staking](/learn/faq/what-are-the-risks-associated-with-staking). 
```

Comment (barriebyron) [reply content/articles/what-is-delegating.md]: good catch

**Pull request merged**